ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml

```diff
@@ -293,6 +293,48 @@
                     current-parameters: true
                   - name: 'ceph-ansible-prs-nautilus-centos-container-add_mons'
                     current-parameters: true
+      - conditional-step:
+          condition-kind: shell
+          condition-command: |
+            #!/bin/bash
+            set -x
+            # if the target branch is master we RUN these tests.
+            if [[ "$ghprbTargetBranch" =~ "stable-" ]]; then
+              exit 1
+            fi
+            git diff --name-only $(git show HEAD | grep Merge | head -n 1 | cut -d ':' -f2) | grep -E 'infrastructure-playbooks/add-mgrs'
```

Comment:
probably need to add '|tests/functional/add-mgrs' there (like add-mons)

Reply:
Oops. I was supposed to keep "tests/functional/add-mgrs" and was supposed to remove "infrastructure-playbooks/add-mgrs". I've fixed it.